### ISSUE TITLE
The --selector flag in kubectl rollout restart not recognized

### ISSUE THREAD

**Issue by Ga13Ou** [kubernetes/kubernetes#112018]
### What happened?

When I use the command 
`kubectl rollout restart deployment --selector "label=value"`
I get this error: unknown flag: --selector

Even though the flag is clearly indicated in the examples of the rollout command in the reference docs

### What did you expect to happen?

A restart of all deployment matching the label

### How can we reproduce it (as minimally and precisely as possible)?

`kubectl rollout restart deployment --selector "label=value"`

### Anything else we need to know?

The documentation link:
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-restart-em-

### Kubernetes version

<details>

```console
$ kubectl version

Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.5", GitCommit:"5c99e2ac2ff9a3c549d9ca665e7bc05a3e18f07e", GitTreeState:"clean", BuildDate:"2021-12-16T08:38:33Z", GoVersion:"go1.16.12", Compiler:"gc", Platform:"darwin/arm64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.13-eks-84b4fe6", GitCommit:"e1318dce57b3e319a2e3fecf343677d1c4d4aa75", GitTreeState:"clean", BuildDate:"2022-06-09T18:22:07Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
```

</details>

### Cloud provider

<details>
AWS
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by SataQiu** [comment]
You can try to update your `kubectl` version, `--selector` is available since v1.24.
FYI: https://github.com/kubernetes/kubernetes/commit/5a5ac86a80464019e3e85d57306156839511d397

**Comment by neolit123** [comment]
/kind support
/close

please see
https://github.com/kubernetes/kubernetes/blob/master/SUPPORT.md
